pkg/compute/rust.go

```diff
@@ -303,13 +303,13 @@ func (r Rust) Build(out io.Writer, verbose bool) error {
 	if verbose {
 		args = append(args, "--verbose")
 	}
-	// Add debuginfo RUSTFLAGS to command environment to ensure DWARF debug
+	// Append debuginfo RUSTFLAGS to command environment to ensure DWARF debug
 	// information (such as, source mappings) are compiled into the binary.
-	env := append(os.Environ(), `RUSTFLAGS=-C debuginfo=2`)
+	os.Setenv("RUSTFLAGS", os.Getenv("RUSTFLAGS")+` -C debuginfo=2`)
```

Comment:
What happens when `RUSTFLAGS` isn't set and so we end up setting `RUSTFLAGS= -C debuginfo=2` with a space after the `=` character?

Reply:
Nothing.